client/src/components/admin/tables/ProjectRow.tsx

```diff
@@ -91,13 +111,39 @@ const ProjectRow = ({ project, idx, checked, handleCheckedChange }: ProjectRowPr
                     ></input>
                 </td>
                 <td>{project.name}</td>
+                <td className="flex justify-center">
+                    <button
+                        onClick={() => {
+                            setShowFlags(true);
+                        }}
+                    >
+                        {matchingFlag ? (
```

Comment:
Why do we need to have a `matchingFlag` variable if we can just check if matchingFlagCount is greater than 0?

Reply:
Also in this same vein, can we just renamed the `matchingFlagCount` state variable to `flagCount`. It makes it more clear when using it in the JSX.